site/plugins.md

```diff
@@ -390,8 +391,257 @@ func (p *Plugin) Report(w http.ResponseWriter, r *http.Request) {
+- `dataType` specifies the type of data, this will determine how the value is displayed. Possible values for this attribute are: "number", "ip", "datetime" and "" for strings.
+- `priority` is a floating point value used to decide the display ordering (lower values are displayed before higher ones). If omitted, the UI will display it last.
+- `from` indicates where to look for the metadata. The possible values are:
+  - `latest`
+  - `sets`
+  - `counters`
+
+### Table Templates
+Table Templates describe a table and how to identify which metadata templates belong to the table.
+
+```json
+"table_templates": {
+      "table-template-id": {
+        "id": "table-template-id",
+        "label": "Human-readable description",
+        "prefix": "table-id-"
+      },
+      "another-table-template-id": {...}
+}
+```
+
+- `table-template-identifier` and `id` identify a particular table template.
+- `label` contains the label that will be used by Scope UI.
+- `prefix` is used to identify which metadata templates belong to the table.
+
+If you want to display data in a table, you have to define a table template and prepend the table prefix to all the metadata templates that identify the data you want to put in such table.
+
+### Metrics
+Metrics are a particular kind of data that can be plotted on the UI as a graph.
+Scope uses the `metric_templates` to know how to display such data. To pair a metric with its template, it is necessary to use the `metric-template-id` as key to identify that particular metric.
+Metrics are suitable for information such as CPU and memory usage, HTTP requests rate, I/O operations, etc.
+The following is an example of a report with a metric preceded by its metric template:
+
+```json
+"metric_templates": {
+      "metric-template-id": {
+        "id": "metric-id",
+        "label": "Human-readable description",
+        "format": "percent",
+        "priority": 1.6
+      },
+}
 
+"metrics": {
+	"metadata-template-id": {
+		"samples": [
+			{
+				"date": "2016-11-17T08:53:03.171424664Z",
+				"value": 98.24
+			},
+			{
+				"date": "2016-11-17T08:53:04.171789887Z",
+				"value": 80.11
+			}
+		],
+		"min": 0,
+		"max": 100
+	}
+}
+```
+
+- `samples` is the list of the samples for this report.
+- `min` is the minimum value possible.
+- `max` is the maximum value possible.
+
+### Metric Templates
+Metric Templates describe a particular metric.
+The following is an example of metric template:
+
+```json
+"metric_templates": {
+      "metric-template-id": {
+        "id": "metric-id",
+        "label": "Human-readable description",
+        "format": "percent",
+        "priority": 1.6
+      },
+      "another-metric-template-id": {...}
+}
+```
+
+- `metric-template-id` and `id` identify a particular metric template.
+- `label` contains the label that will be used by Scope UI.
+- `format` describes how the metrics is formatted.
+  - `percent` the metric value is a percentage.
```

Comment:
`metric-template-id` and `id` identifies a specific metric template.
 `label` contains the label used by Scope UI.
`format` describes how the metrics are formatted. Format options can be: